### DISCUSSION TITLE
Custom App Plugin Redirect Feature

### DISCUSSION THREAD

**Discussion by aishyandapalli** [grafana/grafana#39610] [Plugins]
We don't have redirection for `plugins/:pluginId/page/:slug` yet. It's still not implemented in React. I can contribute to this. I've already implemented the code

**Comment by amovalia** [comment]
@aishyandapalli  I also need this feature, is this possible for you to share fork/sample of code?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
